src/slider/slider.scss

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$thumb-height: $iui-m;  // 16px
+$track-height: $iui-xs;  // 4px
+$tick-height: $iui-baseline; // 11px
+
+@mixin iui-slider-component-container {
+  display: flex;
+
+  .iui-slider-min,
+  .iui-slider-max {
+    @include iui-font-family;
+    user-select: all;
+    margin-top: $iui-xxs;
+
+    .iui-icon {
+      @include iui-icons-default;
+      margin-top: $iui-xs;
+    }
+  }
+
+  .iui-slider-min {
+    margin-right: $iui-sm;
+    text-align: right;
+  }
+
+  .iui-slider-max {
+    margin-left: $iui-sm;
+  }
+
+  &.iui-disabled {
+    @include iui-slider-disabled;
+  }
+}
+
+@mixin iui-slider-container {
+  position: relative;
+  flex-grow: 1;
+  pointer-events: all;
+  cursor: pointer;
+
+  .iui-slider-rail {
+    position: absolute;
+    width: 100%;
+    height: $track-height;
+    top: $iui-sm;
+    @include themed {
+      background-color: t(iui-color-background-4);
+    }
+  }
+
+  .iui-slider-thumb {
+    position: absolute;
+    pointer-events: all;
+    height: $thumb-height;
+    width: $thumb-height;
+    border-radius: 100%;
+    top: $track-height + 1; // position center of thumb in center of track
+    transform: translateX((-$thumb-height - 2) * 0.5); // Adjust the left (set in code) by radius of thumb so thumb center is at value position
+    z-index: 1;
+    cursor: grab;
+    @include themed {
+      background: t(iui-color-background-1);
+      border: 1px solid t(iui-color-background-5);
+    }
+
+    &:active {
+      cursor: grabbing;
+    }
+
+    @include iui-focus();
```

Comment:
I don't think this is working. I'm able to make the focus ring show up when I click.

Reply:
hmm..
Thats interesting.. I tried to have just focus and focus-visible rules like that, it still got the style on click:
![image](https://user-images.githubusercontent.com/81580355/125267374-672dc700-e30f-11eb-87fd-6bfa315ebaba.png)
Not sure, but might be some different case and browser decides that it needs to get the focus?
https://css-tricks.com/almanac/selectors/f/focus-visible/
There is a bunch of articles that sometimes it can be different... (inputs, buttons are the safe bet)